staging/src/k8s.io/kube-scheduler/config/v1alpha2/types.go

```diff
@@ -39,9 +39,6 @@ const (
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// AlgorithmSource specifies the scheduler algorithm source.
```

Comment:
We only allow users to pass in one configuration (either v1alpha1 or v1alpha2). And then we use the internal types to build profiles.

We have a flag to write the configuration to a file and exit the scheduler. This only has debugging purposes (for kube-scheduler devs), but it would be broken with this change when using v1alpha1 (it only outputs in v1alpha2).

Reply:
@lavalamp any additional concerns?